jsonnet/obs-operator.jsonnet

```diff
@@ -415,6 +424,77 @@ local operatorObs = obs {
           },
         },
       } else {}
+    ) + (
+      if (v.kind == 'StatefulSet' &&
+        std.startsWith(v.metadata.name, cr.metadata.name + '-thanos-receive') &&
+        std.objectHas(cr.spec.thanos.receivers, 'containers')) then {
+        template+: {
+          spec+: {
+            containers: override_containers(super.containers, cr.spec.thanos.receivers.containers)
+          },
+        },
+      } else {}
+    ) + (
+      if (v.kind == 'StatefulSet' &&
+        std.startsWith(v.metadata.name, cr.metadata.name + '-thanos-store-shard') &&
+        std.objectHas(cr.spec.thanos.store, 'containers')) then {
+        template+: {
+          spec+: {
+            containers: override_containers(super.containers, cr.spec.thanos.store.containers)
+          },
+        },
+      } else {}
+    ) + (
+      if (v.kind == 'StatefulSet' &&
+        std.startsWith(v.metadata.name, cr.metadata.name + '-thanos-store-memcached') &&
+        std.objectHas(cr.spec.thanos.store.cache, 'containers')) then {
+        template+: {
+          spec+: {
+            containers: override_containers(super.containers, cr.spec.thanos.store.containers)
+          },
+        },
+      } else {}
+    ) + (
+      // Will this also apply to QFE??
```

Comment:
Possibly, as we match with "thanos-query"?

Reply:
oh opps, I forgot about that...